python/metatensor-learn/metatensor/learn/_backend.py

```diff
@@ -9,7 +9,14 @@
 #
 # Any change to this file MUST be also be made to `metatensor/torch/learn.py`.
 
-from metatensor import Labels, LabelsEntry, TensorBlock, TensorMap
+from metatensor import (
+    Labels,
+    LabelsEntry,
+    TensorBlock,
+    TensorMap,
+    equal_metadata,
```

Comment:
it might make sense to have `_backend` re-export everything from `metatensor`/`metatensor.torch`, and then add a handful of other functions as needed? Not necessarily to be done in this PR!